ui/raidboss/data/05-shb/raid/e12n.js

```diff
@@ -1,8 +1,73 @@
+import Conditions from '../../../../../resources/conditions.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import { Responses } from '../../../../../resources/responses.js';
 import ZoneId from '../../../../../resources/zone_id.js';
 
+// TODO: Tether mechanic callouts. Each tether type is used for one primal,
+// so we could just do a collect > analyze > call system based on which tethers are seen.
+// TODO: Handle the EarthShaker bait --> beam intercept mechanic during the intermission.
+// TODO: Math the spawn position of the Titanic Bomb Boulders to call the safe direction like E4s.
+// TODO: Fix the Rapturous Reach trigger so it doesn't double call during the intermission.
+
+
 export default {
   zoneId: ZoneId.EdensPromiseEternity,
   timelineFile: 'e12n.txt',
   triggers: [
+    {
+      id: 'E12N Maleficium',
+      netRegex: NetRegexes.startsUsing({ id: '5872', source: 'Eden\'s Promise', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E12N Formless Judgment',
+      netRegex: NetRegexes.startsUsing({ id: '5873', source: 'Eden\'s Promise' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankCleave(),
+    },
+    {
+      id: 'E12N Boulders Impact',
+      netRegex: NetRegexes.ability({ id: '586E', source: 'Titanic Bomb Boulder', capture: false }),
+      suppressSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Between small bombs',
+        },
+      },
+    },
+    {
+      id: 'E12N Boulders Explosion',
+      netRegex: NetRegexes.ability({ id: '586F', source: 'Titanic Bomb Boulder', capture: false }),
+      suppressSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Move to last explosions',
+        },
+      },
+    },
+    {
+      id: 'E12N Rapturous Reach',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'E12N Diamond Dust',
+      netRegex: NetRegexes.startsUsing({ id: '5864', source: 'Eden\'s Promise', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Spread, don\'t move',
```

Comment:
Is this the pre-spread before ice comes out? If so, should it just use the responses spread?

Reply:
It is. I guess that probably shouldn't be too bad to have two separate triggers for that. Probably alert on `spread()`and info on `stopMoving()`?

Reply:
Yeah.  I think `spread()` for the pre-spread and `stopMoving()` for ice seems reasonable? Although arguably you slide a very short distance here, so I could also see `spread()` twice?

Reply:
I think twice, yes, once on Diamond Dust and once on head markers.